clients/vscode-hlasmplugin/src/configurationsHandler.ts

```diff
@@ -68,7 +63,7 @@ export class ConfigurationsHandler {
                         ConfigurationsHandler.createProcTemplate(workspace).then(uri => vscode.commands.executeCommand("vscode.open", uri));
                     }
                 });
-        if (!p.exists && !e.exists)
+        if (!p.exists && !b.exists && !e.exists)
```

Comment:
Maybe a topic for another PR - convert this to the "smarter" variant that is used in the code actions?

Reply:
Sure, I can look into that afterwards